cmd/gateway-main.go

```diff
@@ -154,7 +163,7 @@ func newGatewayConfig(accessKey, secretKey, region string) error {
 	serverConfig = srvCfg
 	serverConfigMu.Unlock()
 
-	return nil
+	return accessKey, secretKey, nil
```

Comment:
We don't need to return `accessKey, secretKey`  here in `printGatewayStartupMessage` should use similar to how `printServerStartupMessage`  calling `serverConfig.GetCredential()` 

I see that this type restructuring can wait in another PR @poornas - you can keep the code as is and send the one line change that you did in `mustGetGatewayCredsFromEnv()`